mhr_api/src/mhr_api/reports/v2/report_utils.py

```diff
@@ -512,7 +512,11 @@ def format_description(description: str) -> str:
     doc_desc: str = doc_desc.lower().title()
     doc_desc = doc_desc.replace(' Of ', ' of ')
     doc_desc = doc_desc.replace(' To ', ' to ')
+    doc_desc = doc_desc.replace(' Or ', ' or ')
```

Comment:
There is no way to update the description itself?

Reply:
Yes there is, but the MHR API follows the PPR API convention, where the database values are used as is, and the database values are uppercase. The legacy outputs are all uppercase. The case change is for report formatting. The API consumers receive the same descriptions as they do pre-migration.